canvas_modules/common-canvas/src/common-properties/controls/abstract-table.jsx

```diff
@@ -703,6 +706,8 @@ export default class AbstractTable extends React.Component {
 		if (this.props.control.childItem && !this.isReadonlyTable()) {
 			// set to specific size. Exclude this column from resizing.
 			headers.push({ "key": "subpanel", "label": "", "width": TABLE_SUBPANEL_BUTTON_WIDTH, "staticWidth": true });
+		} else if (this.props.control.rowSelection === ROW_SELECTION.SINGLE) {
```

Comment:
Remove `else` and just add `if `condition. 
This code fails if the table has edit icon and delete icon both, there's only 1 table header. 
To test this case, add `"row_selection": "single",` here - https://github.com/elyra-ai/canvas/blob/main/canvas_modules/harness/test_resources/parameterDefs/structurelisteditor_paramDef.json#L973

There are 7 columns but only 6 headers in the table.
![Screenshot 2024-11-04 at 3 08 00 PM](https://github.com/user-attachments/assets/8bf1a1b5-3a3d-4f76-8f49-33b8bc52dae5)